Resources/Prototypes/Roles/Jobs/Fun/visitors_startinggear.yml

```
@@ -1834,12 +1834,13 @@
   id: PirateScoonerAltA
   equipment:
     jumpsuit: ClothingUniformJumpsuitPirate
-    shoes: ClothingShoesBootsLaceup
+    #Imperial Space Pirates: New Horizon; Line Removed: "shoes: ClothingShoesBootsLaceup"
+    shoes: ClothingShoesCaptainPirateBoots
     head: ClothingHeadBandRed
     id: PiratePDA
     back: ClothingBackpackSatchelLeather
     belt: ClothingBeltUtility
-    ears: ClothingHeadsetFreelance
+    #Imperial Space Pirates: New Horizon; Line Removed: "ears: ClothingHeadsetFreelance"
```

bruh не удаляй строчки. комментируй/заменяй внутренность и напиши типо "# string # Imperial Space Pirates: New horizon". как с shoes. а тут просто закомментируй вот так
```suggestion
    # ears: ClothingHeadsetFreelance #Imperial Space Pirates: New Horizon
```